### DISCUSSION TITLE
What to do if GitHub support doesn't provide any feedback or answer on my ticket

### DISCUSSION THREAD

**Discussion by BKyovtorov21** [community/community#121227] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My school account was suspended last month due to my outlook being compromised. I fixed my security issue and requested for a chance to recover my account. So far a **month** has passed and I have received 0 feedback. I tried to start another ticket which just told me that it is a duplicate ( which it was ). 
What should I do? Is it normal to wait that long for feedback?

**Comment by shox8** [comment]
Good

**Reply by shox8** [reply]
✔
